photon-core/src/main/java/org/photonvision/common/util/math/MathUtils.java

```diff
@@ -71,6 +71,33 @@ public static long nanosToMicros(long nanos) {
         return nanos / 1000;
     }
 
+    /**
+     * Constrain a value to only take on certain values. Pick the next-highest allowed value in the
+     * array if in-between.
+     *
+     * @param value value to quantize
+     * @param allowableSteps sorted array of the allowed values
+     * @return quantized value
+     */
+    public static int quantize(int value, int[] allowableSteps) {
+        for (int step : allowableSteps) {
+            if (value <= step) {
+                return step;
+            }
+        }
+        return allowableSteps[allowableSteps.length - 1];
+    }
+
+    public static double limit(double value, double out_min, double out_max) {
```

Comment:
How is this different from https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/MathUtil.html#clamp(double,double,double)

Reply:
it's not, we should use clamp